### PR TITLE
Remove institutions.section_103_message default value

### PR DESCRIPTION
**Pull request by devin-mccurdy**
## Description
Removed default value for the `institutions.section_103_message` column because this will only be set for IHL institutions. Removed test that is no longer required as this default should not be set.

DB migration PR for https://github.com/department-of-veterans-affairs/gibct-data-service/pull/647

[ZenHub Issue](https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/va.gov-team/8095)

## Testing done
Tested locally & QA reviewed

## Screenshots
N/A

## Acceptance criteria
- [x] The "Institution Summary" area of the "Additional Information" section of the institution profile page ONLY displays the "Protection against late VA payments:" indicator for institutions of type IHL.

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by shawkey-daniel — APPROVED** [review]
LGTM

**Review by delli-gatti-michael — APPROVED** [review]

File: db/migrate/20200421134500_remove_default_from_institutions.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+class RemoveDefaultFromInstitutions < ActiveRecord::Migration[5.2]
+  def change
+    change_column_default(:institutions, :section_103_message,
+                          from: 'No information available at this time', to: nil)
```
Comment (johnpaulashenfelter) [inline db/migrate/20200421134500_remove_default_from_institutions.rb:4]: Are you also changing the existing data for this column to `nil`?

Comment (johnpaulashenfelter) [reply db/migrate/20200421134500_remove_default_from_institutions.rb:4]: Looks like this duplicates the migration in #647

**Review by johnpaulashenfelter — COMMENTED** [review]
I think its likely easier just to deal with #647 that includes this

**Comment by devin-mccurdy** [comment]
> I think its likely easier just to deal with #647 that includes this

@johnpaulashenfelter  This is how we've been handling db migration PRs in the past: a separate PR linked and we get approval and merge the migration PR first.

**Review by johnpaulashenfelter — APPROVED** [review]
:shipit:

**Pull request merged**